sdk/translation/azure-ai-translation-text/CHANGELOG.md

```diff
@@ -1,18 +1,20 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (2024-05-21)
 
 ### Features Added
-  - Added support for AAD authentication.
+  - Added support for Entra Id authentication.
 
 ### Breaking Changes
 
-  - All calls to the client using parameter 'content' have been changed to use parameter 'request_body'.
+  - All calls to the client using parameter 'content' have been changed to use parameter 'body'.
   - Users can call methods using just a string type instead of complex objects.
-
-### Bugs Fixed
-
-### Other Changes
+  - get_languages methods were changed to get_supported_languages.
+  - sent_len property was renamed to sent_len.
+  - from_parameter parameter was renamed to from_parameter.
+  - score parameter was renamed to confidence.
+  - from_script parameter was renamed to from_script.
+  - to_script parameter was renamed to _target_langauge_script.
```

Comment:
With the rename back to the REST names, it looks like this section needs update

Reply:
Thank you, fixed.